src/rHelp.ts

```diff
@@ -389,7 +383,12 @@ export class RHelp implements api.HelpPanel, vscode.WebviewPanelSerializer<strin
 	private async getAllAliases(): Promise<Alias[] | null | undefined> {
 		const aliases = await doWithProgress(() => this.aliasProvider.getAllAliases());
 		if(!aliases){
-			void vscode.window.showErrorMessage('Failed to get list of R functions. Make sure that `jsonlite` is installed and r.helpPanel.rpath points to a valid R executable.');
+			void vscode.window.showErrorMessage([
```

Comment:
I think it's better to keep error messages all on one line. It makes it easier to search for the error message in the code.